spec/penthouse/configuration_spec.rb

```diff
@@ -23,7 +23,7 @@
       it 'should raise an ArgumentError' do
         expect do
           described_class.new(migrate_tenants: true, db_schema_file: File.join(File.dirname(__FILE__), "../support/schema.rb"))
-        end.to_not raise_error(ArgumentError)
+        end.to_not raise_error
```

Comment:
Avoids a warning from RSpec stating than an error would cause this to pass, so better to ensure no error is raised